src/nt_users_tool/constants.py

```diff
@@ -0,0 +1,47 @@
+from lib2to3.pgen2.token import NAME
+
+
+BLANK = " "
```

Comment:
Are you sure you need those constants?

Reply:
Was added for "cleanliness" of the code but it is unnecessary. Will be removed.